woocommerce/woocommerce
# Changed textdomain since API is merged into core now.

### All Submissions:






### Changes proposed in this Pull Request:



This PR changes the text domain in newly merged API files from `woocommerce-rest-api` to `woocommerce`. 

### How to test the changes in this Pull Request:

On master:

1. Run command `wp --allow-root i18n make-pot . --exclude=".github,.wordpress-org,bin,sample-data,node_modules,tests" --slug=woocommerce`
2. Copy file `i18n/languages/woocommerce.pot` to a different location.
3. Check out this branch and run the same command as step 1.
4. Compare the two pot files, paths for translations will be changed but all messages will be there (other than any new changes in master).

### Other information:




### Changelog entry

> Dev - Changed text domain to `woocommerce` for REST API files.
